### PR TITLE
Custom Tooltip

### LINKED ISSUE #38: Create in-house v-tooltip directive
We're currently using https://github.com/Akryum/v-tooltip, but that contains a lot of extra functionality we don't use / need and has a breaking TypeScript problem that doesn't seem to be addressed.

Seeing that we only use a tiny part of the library (just the v-tooltip directive) I think it's easier to lift that into our own codebase. Should give us some more flexibility in how it works / looks too

### PR DESCRIPTION
**Pull request by Nitwel**
closes #38 
- [x] Add basic functionality
- [x] Support left, right, top, bottom
- [x] Ask @rijkvanzanten what features he needs
- [x] wait till he responds

### CODE REVIEW

**Comment by rijkvanzanten** [comment]
Last things:

- [ ] Lets make `top` the default instead of bottom
- [ ] Add an `inverted` modifier that will render the tooltip as light color instead of dark
- [ ] Make sure colors use theme variables instead of named colors (in dark mode the tooltips have to be styled differently

**Comment by benhaynes** [comment]
Another thought — these tooltips should have a max-width and have properly style text wrapping. :)

**Comment by rijkvanzanten** [comment]
Lets limit the max amount of lines in the tooltip to 3. We can use a combination of `max-width` and `line-clamp: 3` 🙂

File: src/styles/_tooltip.scss [hunk 1]
```diff
@@ -1,14 +1,25 @@
 #tooltip {
+
 	--arrow-align: 5px;
+	--tooltip-foreground-color: var(--button-primary-foreground-color);
+	--tooltip-background-color: var(--button-primary-background-color-hover);
```
Comment (rijkvanzanten) [inline src/styles/_tooltip.scss]: Lets move the `--tooltip-foreground-color` / bg to the `themes/default.scss` file so it's a little easier to override when setting up dark mode / high contrast etc

**Comment by rijkvanzanten** [comment]
line-clamp's support isn't really there, and I'd rather not rely on unofficial properties like `-webkit-box-orient` in production code. Lets revert that idea

**Pull request merged**